src/atoms/select/Select.tsx

```diff
@@ -214,6 +214,17 @@ export const Select = ({
     }
   }, [options])
 
+  useEffect(() => {
+    if (isOpen) {
+      const container = document.getElementById(
+        'ahp_main_content_with_scroll_hint'
+      )
+      if (container) {
+        container.scrollTo({ top: container.scrollHeight, behavior: 'smooth' })
+      }
```

Comment:
How does this look when the single select is not the last question? Is there a risk that the page scrolls all the way to the bottom then?

Reply:
See https://www.loom.com/share/8315885dc4a64740a4e14a9bc2c13ba7

Reply:
Let me replicate this first 🤔